### PR TITLE
Add logging

### PR DESCRIPTION
**Pull request by DmytryiStriletskyi**
Check `logging.info` meaning and сompleteness.

### CODE REVIEW

File: olga/analytics/views.py [hunk 1]
```diff
@@ -185,7 +201,22 @@ def post(self, request):
...
 
         if self.is_access_token_authorized(access_token):
+
+            logger.info(
+                'edX installation called {0} from {1} sent statistics after authorization'.format(
+                    received_data.get('platform_name'), received_data.get('platform_url')
+                )
+            )
+
+            logger.info(json.dumps(received_data, sort_keys=True, indent=4))
```
Comment (z4y4ts) [inline olga/analytics/views.py]: This should actually be a `logger.debug`

File: olga/analytics/views.py [hunk 2]
```diff
@@ -185,7 +201,22 @@ def post(self, request):
...
+
             self.create_instance_data(received_data, access_token)
+            logger.info('Corresponding data was created in OLGA acceptor database.'.format(access_token))
```
Comment (z4y4ts) [inline olga/analytics/views.py]: `.format(...)` part is useless. Please delete.

File: olga/analytics/views.py [hunk 3]
```diff
@@ -185,7 +201,22 @@ def post(self, request):
...
             return HttpResponse(status=HTTP_201_CREATED)
 
+        logger.info(
+            'edX installation called {0} from {1} is unauthorized member'.format(
```
Comment (z4y4ts) [inline olga/analytics/views.py]: `is` -> `an`

File: olga/analytics/views.py (hunk 1 above)
Comment (DmytryiStriletskyi) [inline olga/analytics/views.py]: Done.

File: olga/analytics/views.py (hunk 2 above)
Comment (DmytryiStriletskyi) [inline olga/analytics/views.py]: Done.

**Pull request merged**